src/main/java/graphics/renderer/QuadRenderBatch.java

```diff
@@ -5,6 +5,16 @@
 import graphics.ShaderDatatype;
 import graphics.Window;
 
+/**
+ * <h1>Azurite</h1>
+ * Used to render one large rectangle that stretches to all corners
+ * of the window. This can be used in shaders to apply effects based
+ * on the position inside this rectangle, such as point-light effects.
+ *

```

Comment:
The quad renderer is used for more than just a single quad across the window. As the name of this class implies, it is able to handle batch rendering for many quads. Every texture that is rendered is rendered on a quad, as well as any rectangle primitives.

Reply:
I thought the render batch used for multiple quads is DefaultRenderBatch; when looking in the constructor of QuadRenderBatch it called the super constructor with `maxBatchSize = 1`. Maybe I'm missing something?